src/index.js

```diff
@@ -3,35 +3,23 @@ import ReactDOM from 'react-dom';
 import './index.css';
 import Charcuterie from 'pages/Charcuterie';
 import reportWebVitals from './utils/reportWebVitals';
-import TextIcon from 'components/TextIcon'
-import Button from 'components/Button'
+import { BrowserRouter, Route, Link, Switch } from "react-router-dom";
+
 
 ReactDOM.render(
   <React.StrictMode>
-    <Charcuterie >
-      <h2>User Icons</h2>
-      <TextIcon textLetter={'a'} size={'small'} color={'#8EDB31'}>small</TextIcon>
-      <TextIcon textLetter={'b'} size={'default'} color={'#31B4DB'}>default</TextIcon>
-      <h2>Buttons</h2>
-      <p>small</p>
-      <Button size={"small"} type={"primary"} label={"primary"} />
-      <Button size={"small"} type={"secondary"} label={"secondary"} />
-      <Button size={"small"} type={"text"} label={"text"} />
-      <p>medium</p>
-      <Button size={"medium"} type={"primary"} label={"primary"} />
-      <Button size={"medium"} type={"secondary"} label={"secondary"} />
-      <Button size={"medium"} type={"text"} label={"text"} />
-      <p>large</p>
-      <Button size={"large"} type={"primary"} label={"primary"} />
-      <Button size={"large"} type={"secondary"} label={"secondary"} />
-      <Button size={"large"} type={"text"} label={"text"} />
-      <p>others</p>
-      <Button label={"default"} />
-      <Button label={"click me!"} onClick={() => { alert("button clicked"); }} />
-      <br />
-      <Button label={"disabled"} disabled />
-      <Button label={"disabled"} type={"secondary"} disabled />
-    </Charcuterie>
+    <BrowserRouter>
+      <Link to="/">Charcuterie</Link>
+      <Link to="/create-session">Create Session</Link>
+      <Switch>
+        <Route path="/create-session">
```

Comment:
we might need to add "exact path=" here, but we can verify and make changes if needed in the create-session PR

Reply:
hmm ya i think we can verify with your PR (but i did a quick check and it works so far)